test.sh

```diff
@@ -14,4 +14,4 @@ export PIPENV_HIDE_EMOJIS=1
 export PIPENV_COLORBLIND=1
 
 [[ $NO_INSTALL -eq "1" ]] || pipenv install --deploy
-pipenv --verbose run behave --show-timings
+pipenv --verbose run behave --tags=~@local-test-only --show-timings
```

Comment:
what does the `~` do in this case? and don't we need quotes in here?

Reply:
`~` not, so it will not run scenarios or features tagged with `@local-test-only`, in this case, it should only skip `db connection` feature

Reply:
thanks @bissenbay